### PR TITLE
Slack alert and hf scp

### PR DESCRIPTION
**Pull request by shot0829**
#### Checklist
- Have you updated the README (yes/no)?
- Have you updated the CHANGELOG (yes/no)?

### CODE REVIEW

File: scripts/cybershake/add_to_mgmt_queue.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
         "--error",
         type=str,
         help="Errors that occurred during the execution of the script.",
+        default=None,
```
Comment (jpa198) [inline scripts/cybershake/add_to_mgmt_queue.py:36]: Optional arguments are None by default?

File: scripts/management/MgmtDB.py [hunk 2]
```diff
@@ -94,6 +96,14 @@ def update_entries_live(
...
                     self._insert_task(cur, realisation_name, process)
                     logger.debug("New task added to the db")
+                    #send light warnings to alert_url
+                    if alert_url != None:
+                        msg = f"fault:{entry.run_name} step:{entry.proc_type} has failed with error:{entry.error}"
+                        r = requests.post(alert_url, json={'text':msg})
+                elif (entry.status == const.Status.failed.value and self.get_retries(process, realisation_name) >= retry_max) and alert_url != None:
+                    #task failed and meet the cap or retries, send a @here to alert_url for higher attention
+                    msg = f"@here fault:{entry.run_name} step:{entry.proc_type} has failed with error:{entry.error} and meet the retry cap"
+                    r = requests.post(alert_url, json={'text':msg})
```
Comment (jpa198) [inline scripts/management/MgmtDB.py:106]: This should be in a separate function. Preferably in a separate file.
MgmtDB shouldn't be sending internet posts.

File: scripts/management/MgmtDB.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import os
 import sqlite3 as sql
 from logging import Logger
+import requests
```
Comment (jpa198) [inline scripts/management/MgmtDB.py:4]: Add requests to the requirements.txt file

File: scripts/cybershake/add_to_mgmt_queue.py (hunk 1 above)
Comment (shot0829) [reply scripts/cybershake/add_to_mgmt_queue.py:36]: yeah, this shouldnt effect anything, it is just for debugging purposes. ( was having some issue when running on old stuffs)

File: scripts/management/MgmtDB.py (hunk 3 above)
Comment (jpa198) [reply scripts/management/MgmtDB.py:4]: Oh god, you actually have to add requests to the docker library...

File: scripts/management/MgmtDB.py (hunk 2 above)
Comment (shot0829) [reply scripts/management/MgmtDB.py:106]: good point. i can move this up an level to queue_monitor

File: scripts/cybershake/queue_monitor.py [hunk 4]
```diff
@@ -29,6 +30,18 @@
...
+def send_alert(msg, alert_url):
+    data = {"text": msg}
+
+    req = urllib.request.Request(alert_url)
+    # create header and data, slack-bot requires json data
+    req.add_header("Content-Type", "application/json; charset=utf-8")
+    jsondata = json.dumps(data)
+    jsondataasbytes = jsondata.encode("utf-8")
+    req.add_header("Content-Length", len(jsondataasbytes))
+    response = urllib.request.urlopen(req, jsondataasbytes)
```
Comment (jpa198) [inline scripts/cybershake/queue_monitor.py:42]: I think this should be moved to its own file, but it's ok for now. 
If we get any more internet communication stuff it should be moved in future.

**Pull request merged**